Cargo.toml

```diff
@@ -28,6 +28,7 @@ authority_manage = { git = "https://github.com/cryptape/cita-common.git", branch
 rustc-hex = "1.0"
 ntp = "0.4.0"
 min-max-heap = "1.2.2"
+parking_lot = "0.6"
```

Comment:
`parking lot` has been released 0.7 for some time

Reply:
To keep the same version with other dependencies.